datalad/support/gitrepo.py

```diff
@@ -3532,6 +3532,16 @@ def save_(self, message=None, paths=None, _status=None, **kwargs):
                         else ('not a Git repository: %s', exc_str(e)),
                         logger=lgr)
                     continue
+                # This mirrors the result structure yielded for
+                # to_stage_submodules below.
+                yield get_status_dict(
```

Comment:
I was on the fence about whether this result should match the structure yielded for modified subdatasets or whether it should be something like `action='add_submodule', type='dataset', ...`.

Reply:
Your choice would have been my preference too.